src/pykeen/evaluation/evaluator.py

```diff
@@ -481,7 +481,8 @@ def evaluate(
     :param model:
         The model to evaluate.
     :param mapped_triples:
-        The triples on which to evaluate.
+        The triples on which to evaluate. The mapped triples should never contain inverse triples - these are created by
```

Comment:
@cthoyt maybe we can even put this into a `warning` block?

Reply:
No I think it's fine here.